.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -25,6 +25,11 @@ body:
           required: true
         - label: I have resolved all warnings from `brew doctor` and that did not fix my problem.
           required: true
+        - label: I have unset `HOMEBREW_{BOTTLE,ARTIFACT}_DOMAIN` and `HOMEBREW_{BREW,CORE}_GIT_REMOTE` and am still able to reproduce this issue.
+          required: true
+  - type: markdown
+    attributes:
+      value: If this bug is related to mirror sites, please refer to https://github.com/Homebrew/discussions/discussions/1917 for more information, and report bugs to respective mirror maintainers.
```

Comment:
I'm also thinking that this part might fit better in [`config.yml`](https://github.com/Homebrew/brew/blob/bdc3391a1319f1c40a108d14a077ca0964b33a3f/.github/ISSUE_TEMPLATE/config.yml).

It might also make sense to add a separate "Mirrors" category to the Discussions page so that it's not all contained in Homebrew/discussions#1917, but we should probably wait and see how much discussion there is in Homebrew/discussions#1917 before trying this.

Reply:
Just took a look at `config.yaml`. This seems to be used to configure this page https://github.com/Homebrew/brew/issues/new/choose. In fact, a lot of users of ours might not even be aware that they are using a mirror until they want to report a bug, from our observation... So it's better to remind things about mirror sites just after "I have unset `HOMEBREW_{BOTTLE,ARTIFACT}_DOMAIN`".

Reply:
Can also go in both

Reply:
Good idea! Will update tomorrow 😴